### PR TITLE
First version

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/createpanelofnormals/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/createpanelofnormals _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ body:
         Before you post this issue, please check the documentation:
 
         - [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
-        - [nf-core/createpanelofnormals pipeline documentation](https://nf-co.re/createpanelofnormals/usage)
+        - [qbic-pipelines/createpanelofnormals pipeline documentation](https://github/qbic-pipelines/createpanelofnormals/docs)
```
Comment (mirpedrol) [inline .github/ISSUE_TEMPLATE/bug_report.yml:11]:
```suggestion
        - [qbic-pipelines/createpanelofnormals pipeline documentation](https://github.com/qbic-pipelines/createpanelofnormals/tree/master/docs)
```

File: assets/schema_input.json [hunk 2]
```diff
@@ -10,27 +10,39 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample ID must be provided and cannot contain spaces",
+                "meta": ["sample"]
```
Comment (mirpedrol) [inline assets/schema_input.json:14]: If you use `"meta": ["id"]` here you won't need to modify the channel [afterwards](https://github.com/qbic-pipelines/createpanelofnormals/pull/1/files#diff-2dfce64501c68086d38cac9e05a1c1b683efacaf9bff552fabcb8f1ae309e05aR59)

File: workflows/createpanelofnormals.nf [hunk 3]
```diff
@@ -63,24 +54,49 @@ workflow CREATEPANELOFNORMALS {
...
+    intervals   = params.intervals ? Channel.fromPath(params.intervals).first() : Channel.empty()
+
+    fasta = fasta.map{ it -> [[id:it.baseName], it]}
+    fai   = fai.map{ it -> [[id:it.baseName], it]}
+    dict  = dict.map{ it -> [[id:it.baseName], it]}
+
+    // // Combine input and intervals for spread and gather strategy
+    input_intervals = input.combine(intervals)
+        // Move num_intervals to meta map and reorganize channel for MUTECT2_PAIRED module
+        .map{ meta, input_list, input_index_list, intervals -> [ meta + [ num_intervals:0 ], input_list, input_index_list, intervals ] }
```
Comment (mirpedrol) [inline workflows/createpanelofnormals.nf:74]: Is `num_intervals` always 0?

**Review by mirpedrol — APPROVED** [review]
LGTM, only a couple of small comments

File: assets/schema_input.json (hunk 2 above)
Comment (FriederikeHanssen) [reply assets/schema_input.json:14]: ah brilliant :)

File: workflows/createpanelofnormals.nf (hunk 3 above)
Comment (FriederikeHanssen) [reply workflows/createpanelofnormals.nf:74]: added scatter gather now after all

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
 <!--
-# nf-core/createpanelofnormals pull request
+# qbic-pipelines/createpanelofnormals pull request
 
-Many thanks for contributing to nf-core/createpanelofnormals!
+Many thanks for contributing to nqbic-pipelines/createpanelofnormals!
```
Comment (FriederikeHanssen) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
Many thanks for contributing to qbic-pipelines/createpanelofnormals!
```

**Pull request merged**